### ISSUE TITLE
State of the animation fix

### ISSUE THREAD

**Issue by codecheck01** [notgoodusename/OsirisAndExtra#174]
@notgoodusename Hello, I just wanted to know what is the current state of the animation fix, I think this is one of the most important feature and should be prioritized.

- What have to be done / implemented.
- What remains to be fixed.
- Enumerates the actual issues.

Also I would like to know the commands/functions to help to debug the animation fix, I really would focus on it.

**Comment by notgoodusename** [comment]
The current localplayer animfix is finished
when it comes to other players, much is needed to be done

**Comment by Navoichyk** [comment]
> @notgoodusename Hello, I just wanted to know what is the current state of the animation fix, I think this is one of the most important feature and should be prioritized.
> 
> * What have to be done / implemented.
> * What remains to be fixed.
> * Enumerates the actual issues.
> 
> Also I would like to know the commands/functions to help to debug the animation fix, I really would focus on it.

Are you hinting towards the esp? If you are, i totally agree. They need to fix the esp and make it where its not so bipolar and you can at least somewhat consistently see people.

**Comment by codecheck01** [comment]
No, the ESP is a whole different issue, I was just talking about the animation fix (syncing up the client animations to the server).

**Comment by Navoichyk** [comment]
> @notgoodusename I think after taking a look at `invalidateBoneCache`, it is not complete.

Ah, well im beginner in cpp and barely know anything so im not even gonna try lol. I've already tried in the past and all i end up is confused

**Comment by codecheck01** [comment]
@notgoodusename It is still correct to call `localPlayer->drawServerHitboxes();` at the end of `Animations::renderStart` (to test the localplayer animation fix)?

**Comment by notgoodusename** [comment]
> @notgoodusename It is still correct to call `localPlayer->drawServerHitboxes();` at the end of `Animations::renderStart` (to test the localplayer animation fix)?

nope, most of the memory patterns are outdated
